.travis.yml

```diff
@@ -5,7 +5,7 @@ services:
 
 env:
   global:
-    - INSP_VERSION=v2.0.27
+    - INSP_VERSION=insp3
```

Comment:
This should be `v3.0.1` instead of insp3.

I usually have the branch name in the Dockerfile, but the stable version in the .travis-ci.yml. This means TravisCI builds the stable version, but local testing works well for latest changes :)